.travis.yml

```diff
@@ -1,4 +1,9 @@
 language: android
+
```

Comment:
The entire coverity scan implementation?
I'll remove it :+1: (shouldn't be on that branch anyways)